cogs/hideout/boost_roles.py

```diff
@@ -0,0 +1,265 @@
+import logging
+from io import BytesIO
+from typing import Optional, Union
+
+import discord
+from discord import app_commands
+from discord.utils import MISSING
+from emoji import is_emoji
+
+from utils import HideoutCog
+
+from . import HideoutManager
+
+Interaction = discord.Interaction[HideoutManager]
+log = logging.getLogger(__name__)
+
+
+class BoostRoles(HideoutCog):
+    @staticmethod
+    async def get_emoji(input: str) -> Optional[Union[str, bytes]]:
+        if is_emoji(input):
+            return input
+
+        partial = discord.PartialEmoji.from_str(input)
+        if partial.is_unicode_emoji():
+            return input
+
+        elif partial.id is None:
+            return None
+
+        return await partial.read()
+
+    @HideoutCog.listener()
+    async def on_member_update(self, before: discord.Member, after: discord.Member):
+        if before.premium_since and not after.premium_since:
+            db = await self.bot.pool.fetchrow("SELECT * FROM booster_roles WHERE user_id = $1", after.id)
+
+            if db is None:
+                return
+
+            role = after.guild.get_role(db["role_id"])
+
+            if role is None:
+                return
+
+            try:
+                await role.delete(reason=f"Nitro Boost Expired: {after}")
+
+            except (discord.HTTPException, discord.Forbidden) as exc:
```

Comment:
Forbidden is a subclass of HTTPException
```suggestion
            except discord.HTTPException as exc:
```